webapp/components/Comment/Comment.vue

```diff
@@ -141,6 +151,10 @@ export default {
     },
   },
   methods: {
+    reply() {
+      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
+      this.$emit('reply', message)
+    },
     checkAnchor(anchor) {
       return `#${this.anchor}` === anchor
     },
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T07:52:48Z" title="Thursday, January 9th 2020, 8:52:48 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -168,6 +176,23 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    answerComment() {
+      const slug =
+        '<a class="mention" href="/profile/' +
+        this.comment.author.id +
+        '" data-mention-id="' +
+        this.comment.author.slug +
+        '" target="_blank" contenteditable="false">@d' +
+        this.comment.author.slug +
+        '</a>'
+      document.querySelector('.editor-content div').focus()
+      if (document.querySelector('.is-empty')) {
+        document.querySelector('.is-empty').innerHTML = slug + ' '
+      } else {
+        const html = document.querySelector('.editor-content').innerHTML
+        document.querySelector('.editor-content div').innerHTML = html + ' ' + slug + ' '
+      }
```


This is really quick and dirty @ogerly . Or I am mistaken?
May we find another way to get the text into the editor?
Like we tried before? 
Even the cursor does not move so that it is set after the pasted text …

What are @mattwr18 @roschaefer and @alina-beck saying?
Any suggestions to do it more elegant?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-10T07:20:29Z" title="Friday, January 10th 2020, 8:20:29 am +01:00">Jan 10, 2020</time>_

---


yes q & d

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-10T19:56:12Z" title="Friday, January 10th 2020, 8:56:12 pm +01:00">Jan 10, 2020</time>_

---


I figured out a less dirty and more robust way to do it: `this.editor.setContent(/* .. */)`

https://github.com/scrumpy/tiptap#editor-methods

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-14T17:39:27Z" title="Tuesday, January 14th 2020, 6:39:27 pm +01:00">Jan 14, 2020</time>_

---


this is the end :)

```
insertReply(message){
      this.editor.commands.mention({ id: message.id, label: message.slug })
    }
```